### PR TITLE
[runtime-security] Use correct path when changing owners of eBPF programs

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Fix path of eBPF programs when changing owner in post install script

### Motivation

Postinstall script fails because the path of eBPF is wrong

### CODE REVIEW

**Review by mx-psi — APPROVED** [review]
LGTM, I have triggered the kitchen tests (`inv pipeline.trigger --git-ref lebauce/fix-agent-postinstall`) to double check that they pass and it should be okay to merge if they do

**Review by bmikaili — APPROVED** [review]
The part to copy the files over looks fine to me.

**Review by albertvaka — APPROVED** [review]

File: omnibus/package-scripts/agent/postinst [hunk 1]
```diff
@@ -69,8 +69,8 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
     fi
 
     # Make the system-probe binary and eBPF programs owned by root
-    chown root:root ${INSTALL_DIR}/embedded/bin/system-probe
-    chown -R root:root ${INSTALL_DIR}/embedded/usr/share/system-probe
+    chown root:root ${INSTALL_DIR}/embedded/bin/system-probe || true
+    chown -R root:root ${INSTALL_DIR}/embedded/share/system-probe/ebpf || true
```
Comment (sunhay) [inline omnibus/package-scripts/agent/postinst:73]: Why are we okay with this operation failing?

**Review by sunhay — APPROVED** [review]
LGTM - just one question

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -682,6 +682,10 @@ run_dogstatsd_arm_size_test:
     - inv -e system-probe.build --go-version=$SYSTEM_PROBE_GO_VERSION --embedded-path=$DATADOG_AGENT_EMBEDDED_PATH
     - inv -e system-probe.test --only-check-bpf-bytes
     - $S3_CP_CMD $SRC_PATH/$SYSTEM_PROBE_BINARIES_DIR/system-probe $S3_ARTIFACTS_URI/system-probe.$ARCH
+    - $S3_CP_CMD $SRC_PATH/pkg/ebpf/c/tracer-ebpf.o $S3_ARTIFACTS_URI/tracer-ebpf.o.$ARCH
+    - $S3_CP_CMD $SRC_PATH/pkg/ebpf/c/tracer-ebpf-debug.o $S3_ARTIFACTS_URI/tracer-ebpf-debug.o.$ARCH
+    - $S3_CP_CMD $SRC_PATH/pkg/security/ebpf/c/runtime-security.o $S3_ARTIFACTS_URI/runtime-security.o.$ARCH
+    - $S3_CP_CMD $SRC_PATH/pkg/security/ebpf/c/runtime-security-syscall-wrapper.o $S3_ARTIFACTS_URI/runtime-security-syscall-wrapper.o.$ARCH
```
Comment (leeavital) [inline .gitlab-ci.yml:688]: In a follow up commit. can we change this so each new `.o` file doesn't require a change in .gitlab-ci.yml ?

Comment (lebauce) [reply .gitlab-ci.yml:688]: :+1:

**Comment by brycekahle** [comment]
CI seems to be failing with:
```
The following shell command exited with status 1:
     $ objcopy --only-keep-debug /opt/datadog-agent/embedded/share/system-probe/ebpf/tracer-ebpf-debug.o /opt/datadog-agent/.debug/opt/datadog-agent/embedded/share/system-probe/ebpf/tracer-ebpf-debug.o.dbg
 Output:
     (nothing)
 Error:
     objcopy: Unable to recognise the format of the input file `/opt/datadog-agent/embedded/share/system-probe/ebpf/tracer-ebpf-debug.o'
```

File: omnibus/package-scripts/agent/postinst (hunk 1 above)
Comment (lebauce) [reply omnibus/package-scripts/agent/postinst]: We are not ok with it :-) Removed

File: omnibus/config/software/datadog-agent-finalize.rb [hunk 3]
```diff
@@ -182,6 +183,10 @@
             strip_exclude("*psycopg2*")
             strip_exclude("*cffi_backend*")
 
+            # Do not strip eBPF programs
+            strip_exclude("*tcp-tracer*")
```
Comment (brycekahle) [inline omnibus/config/software/datadog-agent-finalize.rb:187]: this should just be `tracer`, not `tcp-tracer`

**Pull request merged**